packages/firestore/lite/test/integration.test.ts

```diff
@@ -785,3 +789,96 @@ describe('Query', () => {
     );
   });
 });
+
+describe('equality', () => {
+  it('for collection references', () => {
```

Comment:
Consider adding a test that compares a `DocumentReference` to a `CollectionReference`, which should evaluate to `false`.

Reply:
Done